<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="XYFuncsBox" name="FuncsBox" native="true"/>
      <widget class="XYImageViewer" name="ImageViewer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="XYFuncPanelsBox" name="FuncPanelsBox" native="true"/>
     </widget>
    </item>
    <item>
     <widget class="XYStorageBox" name="StorageBox" native="true"/>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/open.ico</normaloff>:/open.ico</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/restore.ico</normaloff>:/restore.ico</iconset>
   </property>
   <property name="text">
    <string>还原</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/save.ico</normaloff>:/save.ico</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XYFuncsBox</class>
   <extends>QWidget</extends>
   <header>xyfuncsbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XYImageViewer</class>
   <extends>QWidget</extends>
   <header>xyimageviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XYStorageBox</class>
   <extends>QWidget</extends>
   <header>xystoragebox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XYFuncPanelsBox</class>
   <extends>QWidget</extends>
   <header>xyfuncpanelsbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
